dev_requirements.txt

```diff
@@ -1,2 +1,3 @@
 conan==2.1.0
 cmake==3.28.3
+daemon==1.2
```

Comment:
This probably won't have the desired effect for `cgi-slack.py`, which would be run directly by the web server outside of whatever virtualenv is used here. This really is "dev requirements", i.e. things needed for build.

Maybe it's best to just drop `cgi-slack.py` entirely. We (Left Out) used it for a slackbot for a while, but... (a) everyone's moving to Discord and (b) even on Slack that got basically no usage. I think a generic REST API of some kind would make more sense (I think a lot of people just scrape the HTML currently?)

Reply:
I removed it from dev_requirements again. I dont know if just removing cgi-slack.py will have any other effects, so not removing it myself yet.